encoding/protobuf/inbound.go

```diff
@@ -93,6 +93,18 @@ func (o *onewayHandler) HandleOneway(ctx context.Context, transportRequest *tran
 	return o.handleOneway(ctx, request)
 }
 
+type streamHandler struct {
+	handle func(*transport.ServerStream) error
+}
+
+func newStreamHandler(handle func(*transport.ServerStream) error) *streamHandler {
+	return &streamHandler{handle}
```

Comment:
Last thing to do here is to wrap the context with the encoding call API, once that's done we should be good to ship the stack.